### ISSUE TITLE
Wrong axis order in WMTS Capabilities document?

### ISSUE THREAD

**Issue by tomhel** [mapserver/mapcache#144]
Reading the OGC WMTS Specification and OGC Web Services Common Specification it seem that the axis order of ows:BoundingBox should be as defined by the CRS:

> The number of axes included, and the order of these axes, shall be as specified by the referenced CRS.

For CRS 3006 it is defined as North,East (Lat,Lon), but in the capabilities document produced by MapCache ows:BoundingBox is East,North:

```xml
<ows:BoundingBox crs="urn:ogc:def:crs:EPSG:6.3:3006">
  <LowerCorner>-1200000.000000 4305696.000000</LowerCorner>
  <UpperCorner>2994304.000000 8500000.000000</UpperCorner>
</ows:BoundingBox>
```

However, TopLeftCorner has the correct axis order (North,East):

```xml
<TopLeftCorner>8500000.000000 -1200000.000000</TopLeftCorner>
```

Our grid definition looks like this:

```xml
<grid name="3006">
  <srs>EPSG:3006</srs>
  <size>256 256</size>
  <extent>-1200000 4305696 2994304 8500000</extent>
  <units>m</units>
  <resolutions>4096 2048 1024 512 256 128 64 32 16 8 4 2 1 0.5</resolutions>
</grid>
```

Is this a bug or am I missing something?

**Comment by tbonfort** [comment]
Seems like a bug. PRs accepted, you can check the handling of TopLeftCorner to determine if/when to invert axes.

**Comment by tomhel** [comment]
Ok, should be an easy fix. I will create a pull request for this in the next few days.

**Comment by tbonfort** [comment]
closed with #145
